### PR TITLE
Added links to API previews to README.md

### PR DESCRIPTION
**Pull request by TallTed**

### CODE REVIEW

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
I'd like to merge this ASAP... this is editorial and a clear improvement over the current state. Waiting for at least one more CODEOWNER to approve before merging.

**Comment by mkhraisha** [comment]
this should fix #215 point about linking APIs

**Review by mkhraisha — APPROVED** [review]

**Comment by msporny** [comment]
Two approvals by CODEOWNERS, merging.

**Pull request merged**
